src/Control/LDAPDebugController.php

```diff
@@ -43,6 +43,7 @@ class LDAPDebugController extends ContentController
     protected function init()
     {
         parent::init();
+        ini_set('memory_limit', '-1'); // Increase memory limit to max possible
```

Comment:
You could use `Environment::increaseMemoryLimitTo()` to re-use the core wrapper for this method

Reply:
See comment below for this one: https://github.com/silverstripe/silverstripe-ldap/pull/18#discussion_r219886929